plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1679,6 +1683,7 @@ function wc_get_gallery_image_html( $attachment_id, $main_image = false ) {
 			'data-large_image_width'  => esc_attr( $full_src[1] ),
 			'data-large_image_height' => esc_attr( $full_src[2] ),
 			'class'                   => esc_attr( $main_image ? 'wp-post-image' : '' ),
+			'alt'                     => $alt_text,
```

Comment:
Do we want escaping around this, similar to the other attributes above?

Reply:
Good catch @dkotter , somehow I missed it. Thanks!
- [Escape alt attribute](https://github.com/woocommerce/woocommerce/pull/51936/commits/34a65e8184b9276671d6630800dbbf81ec154dbd)